src/main/scala/uk/gov/nationalarchives/tdr/api/http/RouteLogging.scala

```diff
@@ -0,0 +1,32 @@
+package uk.gov.nationalarchives.tdr.api.http
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.server.RouteResult.{Complete, Rejected}
+import akka.http.scaladsl.server.directives.{DebuggingDirectives, LoggingMagnet}
+import akka.http.scaladsl.server.{Directive0, RouteResult}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.Materializer
+import com.typesafe.scalalogging.Logger
+
+object RouteLogging {
+  implicit val actorSystem: ActorSystem = ActorSystem("graphql-server")
+  implicit val materializer: Materializer = Materializer(actorSystem)
+
+  private val logger = Logger(classOf[Routes])
```

Comment:
Should this be `classOf[RouteLogging]` since you've moved this code into its own object?